code/_onclick/hud/screen_objects.dm

```diff
@@ -799,9 +799,11 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/splash)
 		state = HUNGER_STATE_FINE
 		return
 
+	/*
 	if(HAS_TRAIT(hungry, TRAIT_FAT))
 		state = HUNGER_STATE_FAT
 		return
+	*/
```

Comment:
Shouldn't just comment out code, if we want code back we have git to retrieve it. No sense leaving a mess you know?

Reply:
Just figured I'd say that straight for the future

Reply:
Ah it's fine to just delete it? Well, I shall keep that in mind for the future! Thanks much.